### ISSUE TITLE
Ein deutschsprachiges Informationszentrum

### ISSUE THREAD

**Issue by saskiamueller** [decentraland/districts#44]

**Comment by saskiamueller** [comment]
Hallo, 
ich würde gern eine deutschsprachige "Anlaufstelle" in Decentraland aufbauen, die Informationen und "Stadtführungen" anbietet. Ich finde es wichtig, dass wir uns auch in unserer jeweiligen Muttersprache, falls es so etwas in Zukunft noch geben wird, austauschen können. Ich hoffe, es finden sich noch mehr Interessierte, wir würden eine Art Cafe und InformationsZentrum eventuell auch Fortbildungsangebote und Kurse anbieten, in Bezug auf VR Prgrammierung und Blockchain Technologie, vielleicht so etwas wie das Goetheinstitut. Ich selbst habe keinen technologischen Hintergrund, sondern bin einfach nur neugierig. Leider habe ich nicht viel Mana Kapital, vielleicht möchte noch jemand finanzstarkes einsteigen?

**Comment by finom-metall** [comment]
Einen deutschsprachigen Bereich fände ich auf jeden Fall sehr schön und wichtig für Leute, die dem Englisch evtl. noch nicht mächtig sind. Meiner Meinung nach sollte es so eine Anlaufstelle für jede Sprache geben, wobei die wichtigsten Informationen (in verschiedenen Sprachen) um in Decentraland klar zu kommen wahrscheinlich auf Internetseiten zu finden sein wird. 
Ich würde diesen district mit 5 Land und etwas meiner Zeit unterstützen. 
Du solltest jedoch die hier (https://github.com/decentraland/districts/blob/master/README.md) genannten Punkte auf Englisch zu deinem Vorschlag hinzufügen.

**Comment by saskiamueller** [comment]
Title: Germania ( deutschsprachiges Information- und Kontaktzentrum) 
Description: Hier kannst du erst einmal ankommen, dich informieren, Kontakte knüpfen, Ideen entwickeln und zwar auf deutsch. 
Project Lead: saskiamueller, saskiamuemueller@web.de
Team: fabi2803
Goal: Ein deutschsprachiges Informations- und Kontaktzentrum in Decentraland aufbauen
Category: öffentlich
LAND Needed: >10 Land
Startup Costs: Enthusiasmus, Idealismus, Freiwilligkeit und vielleicht 1000€ für den Anfang? 
Die Gesellschaftsform ist noch nicht festgelegt. 
Attachments / Designs / Budgets: Upload any supporting files.

**Comment by saskiamueller** [comment]
Hallo, ich habe Probleme mit der Installation der Metamask, auf dem iPad geht es gar nicht und mein Windowsxp will auch nichts oder wie ich das will. Deswegen konnte ich  noch kein Land kaufen. Fabi, hast du dich schon in die "Schlange" gestellt, um Land zu kaufen. Hast du schon Land? Kann ich auch ohne metamask Land kaufen über mein etherwallet, Mana habe ich schon. Ich glaube nicht, dann muss ich mir noch einen anderen Rechner suchen....

**Comment by saskiamueller** [comment]
Decentraland for dummies! Wäre auch noch ein gutes Projekt!

**Comment by finom-metall** [comment]
Ich habe am ersten Tag Land bei einem district reserviert ohne zu wissen, dass erst mal nur eine Reservierung geht. Jetzt darf ich warten bis es Neuigkeiten gibt.
Soweit ich weiß geht es nur über Metamask.
Lass dir ruhig Zeit, es gibt keinen Grund zur Eile.
Falls du noch Probleme oder Fragen hast, schreib mir im Slack, ich habe dir schon eine Nachricht hinterlassen.

**Comment by Hermses** [comment]
Ja Cool, eine deutschsprachige community bei Decentraland ....ich habe zwar keine ahnung von nichts ..würde sie aber gerne bekommen und mich irgendwo ein stück weit anschließen :D

**Comment by Hermses** [comment]
seit ihr irgendwo aktiv ?

**Comment by saskiamueller** [comment]
Haha, dann bist du hier genau richtig! Wir sind erst wenige und alles läuft ganz locker, Hauptsache irgendwie dabei sein....vielleicht gehst du auch in den Slack? Oder sonst hier...

**Comment by Hermses** [comment]
Ja herrlich, ..wie finde ich euch da wo ?

**Comment by saskiamueller** [comment]
![0bf183f1-4d72-457e-bf4f-a208945a3577](https://user-images.githubusercontent.com/28843015/32404647-e32ab552-c153-11e7-99fa-54f3ef724eb3.png)

**Comment by saskiamueller** [comment]
Du musst in den Decentraland slack gehen...Saschi, Ifabian

**Comment by Hermses** [comment]
sorry, ich habe keinen plan von slack & weis auch nicht wie ich mich da in dem arbeitsbereich anmeld oder was auch immer ? ... hast nen link?

**Comment by saskiamueller** [comment]
Okay, ich auch nicht wirklich...du müsstest dir die slack App runterladen, ist ein Messenger System, dann zu Decentraland....@saschi...Versuch’s mal. Ich hab auch keine Ahnung, man muss sich so durchfummeln. Erzähl mal, wie bist du auf Decentraland gestoßen und was stellst du dir vor?

**Comment by Hermses** [comment]
Ja, können wir uns gern mal drüber unterhalten, wenn wir es irgendwo schaffen uns ganz normal zu schreiben :-D
